website/docs/tutorials/06-deploying/01-cli.md

```diff
@@ -30,6 +30,37 @@ Transaction hash: 0x3c6c...
 https://goerli.voyager.online/contract/0x038cc...
 ```
 
+### Signing a declaration
+
+As of cairo v0.10 you will be able to sign declare transactions. 
+Our CLI will also provide 2 ways of providing the signature:
+
+### 1. Using the default signer from starknet.py - [StarkCurveSigner](https://starknetpy.readthedocs.io/en/latest/signer.html#module-starknet_py.net.signer)
+
+This way requires you to pass a private key (for signing) and account contract's address (to fetch the nonce). 
+
+2 options are used for this:
+- `private-key-path` - a path to the file containing hex-encoded private key
```

Comment:
how to obtain a keypair for signing? I think it would be nice to explain this in our docs or at least refer to source.

Reply:
To be honest i don't even know what to refer to here XD

Reply:
I can link to argentx, that's probably the most universal one, apart from that there is no clear demonstration on how to obtain the keys

Reply:
done